tests/test_packages/functional/package_test/test_regression_report_filter.py

```diff
@@ -15,16 +15,13 @@
 
 import shared
 
-from test_helper.thrift_client_to_db import CCViewerHelper
-
-from test_helper.testlog import info
+from test_utils.thrift_client_to_db import CCViewerHelper
```

Comment:
I'm not sure I would remove all the info printouts. This could help in debugging. There is a 'debug' log in the test_helper.testlog which could be used to print the information (by default these will not be shown not like info).